File: 3rd sem/R/lab assignment/day1/day1p5.R
# Write a R program to print the first 10 Fibonacci numbers(recursion)

fibb <- function(x1, x2, i) {
    if (i == 10) {
        return()
    }
    print(x2)
    fibb(x2, x1 + x2, i + 1)
}

fibb(0, 1, 0)

# Output
# [1] 1
# [1] 1
# [1] 2
# [1] 3
# [1] 5
# [1] 8
# [1] 13
# [1] 21
# [1] 34
# [1] 55

File: 3rd sem/Python/day1/day1p5.py
# Write a python program to swap two numbers

a = int(input("Enter 1st number: "))
b = int(input("Enter 2nd number: "))
print(f"Before Swapping: a = {a} & b = {b}")
temp = a
a = b
b = temp
print(f"After Swapping: a = {a} & b = {b}")

'''
Output
Enter 1st number: 1
Enter 2nd number: 92
Before Swapping: a = 1 & b = 92
After Swapping: a = 92 & b = 1
'''
